airbyte-integrations/bases/base-java-s3/src/main/java/io/airbyte/integrations/destination/s3/S3ConsumerFactory.java

```diff
@@ -108,9 +107,10 @@ private static AirbyteStreamNameNamespacePair toNameNamespacePair(final WriteCon
     return new AirbyteStreamNameNamespacePair(config.getStreamName(), config.getNamespace());
   }
 
-  private CheckedBiConsumer<AirbyteStreamNameNamespacePair, SerializableBuffer, Exception> flushBufferFunction(final BlobStorageOperations storageOperations,
-                                                                                                               final List<WriteConfig> writeConfigs,
-                                                                                                               final ConfiguredAirbyteCatalog catalog) {
+  private FlushBufferFunction flushBufferFunction(
+                                                  final BlobStorageOperations storageOperations,
```

Comment:
Minor nit: to keep consistent to have the arguments start on the same line as the calling function